PublicSDKAPITester/RCPurchasesAPI.m

```diff
@@ -106,7 +106,7 @@ + (void)checkAPI {
         
     // RCPurchasesDelegate
     [p.delegate purchases:p didReceiveUpdatedPurchaserInfo:pi];
-    [p.delegate purchases:p shouldPurchasePromoProduct:skp defermentBlock:^(RCPurchaseCompletedBlock makeDeferredPurchase) {}];
+    [p.delegate purchases:p shouldPurchasePromoProduct:skp defermentBlock:^(void (^ _Nonnull completion)(SKPaymentTransaction * _Nullable transaction, RCPurchaserInfo * _Nullable info, NSError * _Nullable error, BOOL cancelled)) {}];
```

Comment:
let's add line breaks here

Reply:
I had a hard time deciding where to put breaks because it's so huge.

Reply:
✅